worknenjoy/gitpay
# Clientlist component

Clientlist component created and added to welcome page

## Description

> Clientlist component receives img path and link to the project.Displays on welcome page

## Changes

- Component added on Welcomepage 

## Issue

> Who is using Gitpay component on home page #498

## Impacted Area

> Main page

## Steps to test 

- Run project locally and you will see the client list component on welcome page.

## Before
> 
![capture4](https://user-images.githubusercontent.com/34197506/80694507-464bce00-8add-11ea-8c69-1ceebe1a2299.jpg)


## After
> 
![Capture3](https://user-images.githubusercontent.com/34197506/80694229-de958300-8adc-11ea-81aa-bfc5ac7dd999.JPG)
